src/enemies/health.rs

```diff
@@ -23,16 +23,16 @@ fn kill_enemy(mut commands: Commands, mut events: EventReader<CollisionEvent>) {
             let (entity_1, entity_2) = event.rigid_body_entities();
             let (layers_1, layers_2) = event.collision_layers();
             if is_player_attack(layers_1) && is_enemy(layers_2) {
-                Some([entity_2, entity_1])
+                Some((entity_2, entity_1))
```

Comment:
In rust, it's generally better to use tuples over arrays for that kind of circumstances